### ISSUE TITLE
DateTime Parsing Error when calling getCompanyFundamentals from Java/Scala

### ISSUE THREAD

**Issue by sanjayvenkat2000** [intrinio/java-sdk#1]
To recreate...
```
	val defaultClient: ApiClient = Configuration.getDefaultApiClient
	val auth: ApiKeyAuth = defaultClient.getAuthentication("ApiKeyAuth").asInstanceOf[ApiKeyAuth]
	auth.setApiKey(API_KEY)

	val companyApi = new CompanyApi()
	// Get the fundamentals for a company using its ticker from CompanySummary object
	val fundamentals = companyApi.getCompanyFundamentals("AAPL", null)
	println(fundamentals)
```
Throws
```
Exception in thread "main" org.threeten.bp.format.DateTimeParseException: Text '2018-11-05T08:01:40.000Z' could not be parsed, unparsed text found at index 10
	at org.threeten.bp.format.DateTimeFormatter.parseToBuilder(DateTimeFormatter.java:1590)
	at org.threeten.bp.format.DateTimeFormatter.parse(DateTimeFormatter.java:1491)
	at org.threeten.bp.LocalDate.parse(LocalDate.java:367)
	at com.intrinio.invoker.JSON$LocalDateTypeAdapter.read(JSON.java:213)
	at com.intrinio.invoker.JSON$LocalDateTypeAdapter.read(JSON.java:180)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:82)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220)
	at com.google.gson.Gson.fromJson(Gson.java:887)
	at com.google.gson.Gson.fromJson(Gson.java:852)
	at com.google.gson.Gson.fromJson(Gson.java:801)
	at com.intrinio.invoker.JSON.deserialize(JSON.java:122)
	at com.intrinio.invoker.ApiClient.deserialize(ApiClient.java:701)
	at com.intrinio.invoker.ApiClient.handleResponse(ApiClient.java:904)
	at com.intrinio.invoker.ApiClient.execute(ApiClient.java:831)
	at com.intrinio.api.CompanyApi.getCompanyFundamentalsWithHttpInfo(CompanyApi.java:1235)
```

The error look like the API is expecting a date object and the service is returning a DateTime.

Please advice on how to fix or if there are other flags I need to set.

Thanks

**Comment by kgmiller** [comment]
Thanks @sanjayvenkat2000 - looking into this.

**Comment by kgmiller** [comment]
Hi @sanjayvenkat2000 - this issue is now resolved in the latest release v1.1.2 .
